applications/parami-protocol.md

```diff
@@ -146,29 +146,65 @@ Yanping YANG
-| 0c. | Tutorial | We will provide a standalone tutorial for endusers and developers. This will be a step-by-step guide. |
-| 0d. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests |
-| 0b. | Testnet | Setup and run a testnet |
-| 1. | Substrate module: did | We will create a Substrate module that will generate and manage on-chain DID with detailed preference profile. A referral system will be integrated with DID |
-| 2. | Substrate module: ads | We will create a Substrate module that will manage ads metadata, including traditional ads metadata and rewarding settings |
-| 3. | Substrate module: prices | We will create a Substrate module that will act as the price oracle to provide real-world prices of different kinds of assets |
-| 4. | Substrate chain | All above modules of our custom chain will interact to provide a useable Ad platform. There will be APIs to setup DIDs and ads and APIs that allow transfer assets via DID |
-| 5. | Web UI: Advertiser App | We will create a web-ui for advertisers, hiding the raw APIs from advertisers, providing an easy access to the ad management system |
-| 6. | Wallet App | We will create a web-based wallet that is embedded to WeChat MiniProgram, the tech stack will be vue.js |
-| 7. | Social Integration SDK | We will integrate ads to some social platforms like WeChat, allowing ad viewers to be verified as DID owners and gain rewards. To enable the combination of Wallet App and social platforms, we will rely on the open platform functionality provided by social platforms. The functionality will include DID registry, DID read, Ad display, Ad track and Ad data upload. |
+| 0b. | Documentation | We will provide inline documentation of the code and API documentation to help developers to integrate Parami DID into their project |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article or medium that explains what was done as part of the grant). |
+| 1. | Pallet(s)/did | Build Parami ParaChain based on Substrate3.0.0, the core modules that have been developed include DID(Decentralized Identity) pallet, Decentralised identity is an emerging concept that gives back control of identity to consumers. |
+| 2. | Pallet(s)/bridge | Completed the bridge development between ETH ERC20 and Parami chain. Please refer to https://github.com/parami-protocol/parami/tree/main/pallets/bridge  |
+| 3. | Pallet(s)/nft | Complete NFT pallets, please refer to https://github.com/parami-protocol/parami/tree/main/pallets/nft |
```

Comment:
There's no mention of either of these anywhere else in the application, and to be honest I don't think we're interested to fund either of them (there are already plenty of implementations for both).